### PR TITLE
Samsung.GalaxyBudsManagerExe version 2.1.220907.51

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99331)

### CODE REVIEW

File: manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.GalaxyBudsManagerExe
+PackageVersion: 2.1.220907.51
+PackageLocale: en-US
+Publisher: Samsung Electronics Co, Ltd.
+PackageName: GalaxyBudsManager
+License: Proprietary
+Copyright: Copyright Â© 2019 Samsung
```
[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml:10]: From the "About PC Software" link at the bottom of the application: 
image

File: manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.GalaxyBudsManagerExe
```
Comment (stephengillie) [inline manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.installer.yaml:4]:
```suggestion
PackageIdentifier: Samsung.GalaxyBudsManager
```

File: manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.GalaxyBudsManagerExe
```
Comment (stephengillie) [inline manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Samsung.GalaxyBudsManager
```

Comment (stephengillie) [reply manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml:4]: Please update file paths to match this change.

File: manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.GalaxyBudsManagerExe
```
Comment (stephengillie) [inline manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.yaml:4]:
```suggestion
PackageIdentifier: Samsung.GalaxyBudsManager
```

File: manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.GalaxyBudsManagerExe
+PackageVersion: 2.1.220907.51
+PackageLocale: en-US
+Publisher: Samsung Electronics Co, Ltd.
+PackageName: GalaxyBudsManager
```
Comment (stephengillie) [inline manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.samsung.com/
PublisherSupportUrl: https://www.samsung.com/ca/support/
PrivacyUrl:   https://www.samsung.com/ca/sustainability/security-and-privacy/
PackageName: Galaxy Buds Manager
PackageUrl: https://www.samsung.com/ca/support/mobile-devices/galaxy-buds-install-galaxy-wearable/
```

**Comment by stephengillie** [comment]
Validation is failing because the file paths haven't been updated: 
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\Samsung\GalaxyBudsManagerExe\2.1.220907.51', expected 'manifests\s\Samsung\GalaxyBudsManager\2.1.220907.51'

**Comment by upintheairsheep** [comment]
I added Exe because I realized a microsoft store version already existed.

**Comment by Trenly** [comment]
> I added Exe because I realized a microsoft store version already existed.

The different source names will differentiate it anyways. Also, the store has the Package Identifier as the store application ID, so there wouldn't be any confusion. I would remove the `exe` and leave it just as `Samsung.GalaxyBudsManager`

File: manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.GalaxyBudsManager
+PackageVersion: 2.1.220907.51
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=7238616&CDCttType=SW&ModelType=C&ModelName=SM-R170NZYAXAR&VPath=SW/202211/20221115161021182/GalaxyBudsManager_2.1.220907.51.exe
```
Comment (Trenly) [inline manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/202211/20221115161021182/GalaxyBudsManager_2.1.220907.51.exe
```

File: manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+PackageLocale: en-US
+Publisher: Samsung Electronics Co, Ltd.
+PublisherUrl: https://www.samsung.com/
+PublisherSupportUrl: https://www.samsung.com/ca/support/
+PrivacyUrl:   https://www.samsung.com/ca/sustainability/security-and-privacy/
+PackageName: Galaxy Buds Manager
+PackageUrl: https://www.samsung.com/ca/support/mobile-devices/galaxy-buds-install-galaxy-wearable/
+License: Proprietary
+Copyright: Copyright © 2019 Samsung Electronics Co., Ltd. All rights reserved.
+ShortDescription: GalaxyBudsManager 2.1.220907.51
```
Comment (Trenly) [inline manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml:15]: Please provide a better description of what GalaxyBudsManager is used for / does

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**